Automattic/wp-calypso
# Site Settings: Add Jetpack Firewall Toggle to Security Settings Screen

Related to https://github.com/Automattic/jetpack-scan-team/issues/1228

Depends on https://github.com/Automattic/jetpack-scan-team/issues/1321

## Proposed Changes

* Adds a "Web Application Firewall (WAF)" setting card to `/settings/security/[site]`.
* If the user does not have access to automatic rules, an upgrade prompt is displayed.
* Merges the Brute Force Protection and IP Allow List settings into the new Firewall card.

## Why are these changes being made?


* See peb6dq-2v0-p2

## Testing Instructions



- Check out this PR in your local Jetpack repo: https://github.com/Automattic/jetpack/pull/38580
- Boot up your local monorepo site with `jetpack docker up -d` and `jetpack docker jt-up`
- Visit your site's `wp-admin` and ensure Jetpack is connected
- Test this PR using the Calypso live link below, and your local test site running through Jurassic Tube.

## Pre-merge Checklist




## Screenshots

<img width="745" alt="Screenshot 2024-07-23 at 6 01 43 PM" src="https://github.com/user-attachments/assets/67d843fd-2fcc-419e-bfa3-b1b5e2973d1e">

<img width="747" alt="Screenshot 2024-07-23 at 6 03 52 PM" src="https://github.com/user-attachments/assets/a3219115-4bec-414b-8400-ac64564396cd">
